pages/linux/zgrep.md

```diff
@@ -0,0 +1,32 @@
+# zgrep
+
+> Grep pattern(s) of text (strings/regex) from files within compressed file (equivalent to grep -Z).
+> Syntax: zgrep [grep_options] pattern file.
+
+- Grep pattern from compressed file (case-sensitive):
+
+`zgrep pattern path/to/compressed/file`
```

Comment:
```suggestion
`zgrep {{pattern}} {{path/to/compressed_file}}`
```
Missing braces for variable arguments. Should be similar in other examples too.

Reply:
changed, only line I wasnt sure about was line 23, I asked in gitter/tldr room. But used my best judgement, please check line 23